core/src/main/java/org/radargun/Slave.java

```diff
@@ -93,7 +93,12 @@ public static void main(String[] args) {
       Slave slave = new Slave(new RemoteMasterConnection(ArgsHolder.getMasterHost(), ArgsHolder.getMasterPort()));
       try {
          slave.run(ArgsHolder.getSlaveIndex());
+      } catch (IOException e) {
```

Comment:
Your printing the exception out, so I guess this could be collapsed to `catch (Exception e)` (without catching IOException).

Reply:
Good point!